### PR TITLE
Auth Create

### PR DESCRIPTION
**Pull request by PraveenSFW**

### CODE REVIEW

File: src/auth/constants.ts
Comment (santhosh-sfw) [inline src/auth/constants.ts:1]: Can you please explain this

**Review by santhosh-sfw — CHANGES_REQUESTED** [review]
Please check the review points

File: src/auth/constants.ts
Comment (PraveenSFW) [reply src/auth/constants.ts:1]: sir, this is a secret private key file for Jwt Auth.

Comment (santhosh-sfw) [reply src/auth/constants.ts:1]: Private key should be generate from node cipher text. Not like this. Please refer
https://stackoverflow.com/questions/6953286/how-to-encrypt-data-that-needs-to-be-decrypted-in-node-js

**Pull request merged**